eng/publish-coverlet-result-files.yml

```diff
@@ -0,0 +1,44 @@
+steps:
+- task: Powershell@2
+  displayName: Prepare coverage files to Upload
+  inputs:
+    targetType: inline
+    pwsh: true
+    script: |
+      New-Item -ItemType Directory "$(Build.SourcesDirectory)/artifacts/TestLogs/"
+      function Copy-FileKeepPath {
+      param (
+          $filter,$FileToCopy,$des,$startIndex
```

Comment:
Can we stay consistent with the naming of the parameters. I think the default in powershell is capitalized e.g. `$Filter, $FileToCopy, $Destination, $StartIndex`.